src/tls.c

```diff
@@ -8098,6 +8098,17 @@ static void findEccPqc(int *ecc, int *pqc, int group)
     }
 }
 
+#if defined(WOLFSSL_MLKEM_CACHE_A) && \
```

Comment:
Please add some inline comments for use of these new macros.

Reply:
Added comment about what this does.